homeassistant/components/unifi_direct/device_tracker.py

```diff
@@ -67,11 +67,11 @@ def update_clients(self) -> bool:
         """Update the client info from AP."""
         try:
             self.clients = self.ap.get_clients()
-        except UniFiAPConnectionException:
-            _LOGGER.error("Failed to connect to accesspoint")
+        except UniFiAPConnectionException as e:
+            _LOGGER.error("Failed to connect to accesspoint: %s", str(e))
```

Comment:
I know it's a very small change, but the "perfect" PR recommendation indicates this should be a separate PR from the dependencency bump
See point 7 in https://developers.home-assistant.io/docs/review-process?_highlight=perfect#creating-the-perfect-pr

Reply:
okay I have removed it and created a seperate PR